.github/workflows/changelog_verifier.yml

```diff
@@ -14,6 +14,8 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
 
       - uses: dangoslen/dependabot-changelog-helper@v1
+        with:
+          activationLabel: 'dependencies'
```

Comment:
This [label is generic](https://github.com/opensearch-project/OpenSearch/pulls?q=is%3Apr+label%3Adependencies) and is used by other PRs as well. 
We could potentially introduce a new label for dependabot to make things easier.

Reply:
@kotwanikunal You can add different labels to Dependabot PRs using https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#labels if this is going to cause conflicts with other PRs with the same labels.

Reply:
Thanks for the suggestion. I was hoping to start a discussion. 
Created a new label `dependabot`. I will update the workflow with the same.

Reply:
PR here: #4348

Reply:
Removed this since the plugin defaults to `dependabot` as the activation label.